lib/src/data/communication/api/network_request.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:allthenews/src/data/communication/api/request.dart';
+import 'package:allthenews/src/di/injector.dart';
+import 'package:allthenews/src/domain/authorization/api_key_repository.dart';
+import 'package:dio/dio.dart';
+import 'package:flutter/foundation.dart';
+import 'package:pretty_dio_logger/pretty_dio_logger.dart';
+
+abstract class _Constants {
+  static const logPrintWidthSize = 300;
+}
+
+class NetworkRequest {
```

Comment:
tu nazwę bym zmienił na jakiś HttpClient bo po nazwie NetworkRequest spodziewałbym się czegoś podobnego do klasy Request ;)

Reply:
też mi to nie pasowało